jira_integration/jira_integration.php

```diff
@@ -0,0 +1,111 @@
+<?php
+// TODO: this header should not be required...
+header('Content-Type: text/plain; charset=UTF-8');
+
+// Only watch the dev environment
+if ($_ENV['PANTHEON_ENVIRONMENT'] != 'dev') {
```

Comment:
Yes. On the flip side, some commits might go multidev -> test -> live, and never go back to dev (some customers use named multidev instead of dev).

Ideally, though, we want to know if a commit has been seen before, and not process it more than once. Might be able to query Jira for all of the comments on the issue, and look for existing references to this commit (maybe slow &/or awkward). Might be able to ignore commits that git reports exists on more than one branch. We'd have to query for all of the multidev branch names, though. You can't push more than one branch at a time with git, right? Are we guaranteed to get a QS hook for one branch before the next one is processed?  Could miss commits otherwise, with that algorithm. Not sure what's best.